docs/content/guides/author-apps/aws/howto-aws-resources/snippets/dynamodb.bicep

```diff
@@ -0,0 +1,34 @@
+import aws as aws
+
```

Comment:
Would there be an opportunity in this guide on showing how a user could access this dynamodb table from a container? Would that require an IAM user and roles and be complicated to setup? If so we may want to start prioritizing connections to AWS resources so we can complete the scenario of this guide. Without information on how to access this resource users will finish this up and not know fully how to proceed.